indra/cmake/Assimp.cmake

```diff
@@ -0,0 +1,19 @@
+# -*- cmake -*-
+
+include_guard()
+
+include(Prebuilt)
+
+add_library( ll::assimp-vc143-mt INTERFACE IMPORTED )
+
+use_system_binary( assimp-vc143-mt )
+
+use_prebuilt_binary(assimp-vc143-mt)
+if (WINDOWS)
+  target_link_libraries( ll::assimp-vc143-mt INTERFACE assimp-vc143-mt)
+elseif (LINUX)
+  target_link_libraries( ll::assimp-vc143-mt INTERFACE assimp-vc143-mt)
+elseif (DARWIN)
+  target_link_libraries( ll::assimp-vc143-mt INTERFACE libassimp-vc143-mt.dylib)
```

Comment:
Quite probably!  I am usually our team's mac developer and I haven't built on mac since inheriting this project and have been focused on the mathematics.
Appreciate the catch.